### DISCUSSION TITLE
在 HttpRunner 永久保持开源免费的情况下，你最期望获得的增值服务有哪些？

### DISCUSSION THREAD

**Discussion by debugtalk** [httprunner/httprunner#1181] [0 - 公告（Announcements）]
HttpRunner 承诺会永久保持开源免费，但同时会考虑提供一些商业化的增值服务。

期望大家能多多理解，在方便的时候也期待大家能给予支持，毕竟纯粹的靠爱发电是无法长久的，能找到一些利益平衡点，互惠互利才能走得更远！

关于增值服务的形式，这里发起一次问卷调查，`只有一个选择题`，麻烦大家动动小手哇，问卷填写数达到 300 后我会将结果发布出来。

再次感谢大家的理解和支持！

问卷链接：https://wj.qq.com/s2/9708369/b6b2/

**Comment by qkp394349328** [comment]
1.希望尽快完成RPC协议的接口测试模块
2.希望能够组建更多的用户交流群，主要是针对刚接触httprunner的这一群体，减少他们使用的学习成本，这样也能给你们提出更优质的建议

**Comment by yanlanfeng** [comment]
1、基于Web接口自动化平台优化一个分支，使其成为一个比较具有独立性、扩展性、特定性但不再基于本地的用例文件运行的插件或库。而是提供特定的接口函数来运行。通过企业增值付费购买后基于此来搭建Web的自动化平台，前端功能让企业自行开发，而让httprunner在后端提供接口可运行数据库存储的用例并返回结果，同时具有接口自动化测试的一些特性或企业自定义功能来满足企业的接口自动化需求。

**Reply by debugtalk** [reply]
@yanlanfeng 你们公司有这样的付费需求么？
